### DISCUSSION TITLE
Ability to control access by the Team

### DISCUSSION THREAD

**Discussion by dberkovich01** [microsoft/vsts-extension-retrospectives#309] [Ideas]
We have multiple Teams working under the same project. Today, anyone from any of these teams can see each other Retrospectives.

In order to make Retrospective a "safe environment", we really would love to see an ability to limit access to Retrospective notes only to Teams that you are a member of.

Thanks!

**Comment by Enterprise-Agile-Coach-C** [comment]
This is a critical need.  Teams need to trust what happens in their retro, stays in their retro.  Not all Teams are preforming, some are storming or have sensitive topics they want to work through.  Please enable this feature.

**Comment by dphanson63** [comment]
This was implemented in a release earlier this year.
